crnk-operations/src/main/java/io/crnk/operations/server/OperationsModule.java

```diff
@@ -263,8 +263,8 @@ private boolean bulkExecuteOperations(List<OrderedOperation> operations, Operati
             JsonPath repositoryPath = new ResourcePath(rootEntry, null);
 
             PreconditionUtil.assertTrue(
```

Comment:
I don't think it is that easy... But glad to be proof  wrong on this. There is a BulkClientTest where patch-related test could be added.

Reply:
What I don't quite understand is why `BulkResourceRepository` conflicts with the operations module. Because bulk `PATCH` requests can work via either option, just not both simultaneously.

Reply:
the operations module is a bit more generic, it can take any number of changes. While the buk patch will "just" take multiple updates from a single type. Ideally the operations module will make use of bulk repositories where available. in what sense you mean "conflict"? 

just now that part of the code base is also not entirely in my mind. So I'm not quite sure where problem manifest. One issue is probably the URL handling. PATCH request assume /api/{resource}/{id} whereas in the bulk case it can only be /api/{resource} and all the IDs are then extracted from the body. Something to account for in `ResourcePatchController`

Reply:
I described the issue a bit better here: https://github.com/crnk-project/crnk-framework/issues/761

Essentially, implementing `BulkResourceRepository` for a given resource effectively breaks operations module `PATCH` requests for that resource. But for the same resource when not implementing `BulkResourceRepository`, `PATCH` requests via the operations module work as expected.

Reply:
yes, let us  just remove that check. Add a test case and fix whatever comes up.